libc/config/config.json

```diff
@@ -21,6 +21,10 @@
     "LIBC_CONF_STRING_UNSAFE_WIDE_READ": {
       "value": false,
       "doc": "Read more than a byte at a time to perform byte-string operations like strlen."
+    },
+    "LIBC_CONF_MEMSET_X86_USE_SOFTWARE_PREFETCHING": {
+      "value": false,
+      "doc": "Use software prefetching in memset to increase performance."
```

Comment:
Let's make it as precise as possible since it serves as documentation.
Something like "Inserts prefetch for write instructions (PREFETCHW) for memset on x86 to recover performance when hardware prefetcher is disabled."